libc-bottom-half/headers/public/wasi/wasip2.h

```diff
@@ -1084,7 +1084,7 @@ extern bool environment_initial_cwd(wasip2_string_t *ret);
 
 // Imported Functions from `wasi:cli/exit@0.2.0`
 // Exit the current instance and any linked instances.
-extern void exit_exit(exit_result_void_void_t *status);
+_Noreturn extern void exit_exit(exit_result_void_void_t *status);
```

Comment:
Since editing these files is unexpected, but I agree necessary in this case, please add to the comment here noting this signature has been edited from the generated code. I'm not sure if anyone will ever read the comment, but it might make it stick out more in a git diff if the files get regenerated in the future.